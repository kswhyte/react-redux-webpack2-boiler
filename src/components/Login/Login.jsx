import React from 'react';

import './login.css';

const Login = () => {
    return (
        <div className="login-block">
            <h1>Login</h1>
        </div>
    )
}

Login.propTypes = {};

export default Login;