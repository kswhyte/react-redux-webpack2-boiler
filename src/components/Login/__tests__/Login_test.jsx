import React from 'react';
// import { storiesOf, action, linkTo } from '@kadira/storybook';
import Login from '../Login';

// import { specs, describe, it } from 'storybook-addon-specifications';

import {storiesOf, describe, it, specs
} from "../../../../.storybook/facade";

import {shallow} from 'enzyme';
import expect from 'expect';


const stories = storiesOf('Login', module);

stories.add('Search Patient Story', () => {
    const loginStory = (
      <Login/>);

    specs(() => describe('Show a successful alert', () => {
        it('Should render the Login component without crashing', () => {
            let output = shallow(loginStory);
            expect(output.find('.login-block').length).toEqual(1);
        });
    }));
    return loginStory;
});
