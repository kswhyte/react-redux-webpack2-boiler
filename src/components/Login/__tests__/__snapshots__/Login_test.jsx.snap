// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Search Patient Story 1`] = `
<div
  className="login-block"
>
  <h1>
    Login
  </h1>
</div>
`;
